pkg/signature/kms/cliplugin/common/interface.go

```diff
@@ -85,3 +88,13 @@ type CreateKeyArgs struct {
 type CreateKeyResp struct {
 	PublicKeyPEM []byte `json:"publicKeyPEM"`
 }
+
+// SignMessageArgs contains the serialized arguments for `SignMessage()`.
+type SignMessageArgs struct {
```

Comment:
Does this need a `CtxDeadline`? Same with `DefaultAlgorithmArgs`

Reply:
No, because in `SignerVerifier`, this method's context is in its opts argument.